### DISCUSSION TITLE
Dark Mode Toggle Switch

### DISCUSSION THREAD

**Discussion by VaibhavSharma24** [Kong/kong#11929] [Ideas and feature requests]
I want to add a dark mode toggle switch on your website it will make this amazing website more convenient to surf through especially in dark.

**Comment by StarlightIbuki** [comment]
Thanks for the advice. Could you elaborate? I'm not sure if I understand it correctly, do you mean Kong Manager pages, online documents, or the official site main page by "website"?
If you want to improve Kong Manager, we are open to community PRs.
Also, to get a dark mode experience we could try browser plugins: https://chrome.google.com/webstore/detail/dark-mode/dmghijelimhndkbmpgbldicpogfkceaj
